parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -355,8 +361,10 @@ async def folder_create(self, path: FsPath) -> EntryID:
             await self.local_storage.set_manifest(parent.id, new_parent)
 
         # Send events
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=child.id)
+        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id, reason=None)
```

Comment:
no reason is a bad reason !
We'd better have `reason=FSEntryUpdatedReason.FOLDER_CREATE` here (to be coherent with the fact `reason=FSEntryUpdatedReason.FOLDER_DELETE` is triggered for the parent)

Of cours this event should provide `entry_id` and `entry_name`